joomla/joomla-cms
# [4.0] Using ListField for fields already namespaced

### Summary of Changes
As title says. Changing `extends \JFormFieldList` to `extends ListField`
Therefore also deleting `FormHelper::loadFieldClass('list');` and `use Joomla\CMS\Form\FormHelper;`


### Testing Instructions

Apply patch and check that everything is still working fine in admin.

@wilsonge
